Flutter iOS symbolication (#5537)
getsentry/sentry-docs: src/platforms/flutter/troubleshooting.mdx

## Known Limitations

- If you enable the `split-debug-info` feature, you must upload the Debug Symbols manually.

> - Issue's titles might be obfuscated as we rely on the `runtimeType`, but they may not be human-readable, See: [Obfuscate Caveat](https://flutter.dev/docs/deployment/obfuscate#caveat)

Reviewer: ```suggestion
- Issue titles might be obfuscated as we rely on the `runtimeType`, but they may not be human-readable. See the [Obfuscate Caveat](https://flutter.dev/docs/deployment/obfuscate#caveat).
```

## Native Symbolication on Android

The configuration for symbolication of Native events (C/C++) is documented in our [Android Native Development Kit](/platforms/android/using-ndk/) content. If you are having issues with symbolication in Flutter, check that your configuration is correct, as discussed in our Flutter content that covers [Uploading for Android NDK](/platforms/flutter/upload-debug/#uploading-for-android-ndk)

## Native Symbolication on iOS/macOS

> Flutter `split-debug-info` and `obfuscate` flags are supported on iOS/macOS; it requires compiling your app using the latest Flutter's `master` branch and the Sentry Flutter SDK `6.10.0` and above.

Reviewer: ```suggestion
Flutter `split-debug-info` and `obfuscate` flags are supported on iOS/macOS. They require compiling your app using the latest Flutter `master` branch and the Sentry Flutter SDK, version `6.10.0` and above.
```
